### PR TITLE
OCPBUGS-16220: Replace Ublox with direct call to default gpsd  for monitoring and  reduce CPU usage 

### PR DESCRIPTION
**Pull request by aneeshkp**
This PR uses the direct call to gpsd for monitoring the GNSS state.
Current monitoring doesn't have GNSS offset. 
Once it is available then the offset will be reported accurately.

### CODE REVIEW

File: pkg/gpsd/gpsd.go [hunk 1]
```diff
@@ -0,0 +1,317 @@
+package gpsd
```
Comment (josephdrichard) [inline pkg/gpsd/gpsd.go:1]: This is an external package?  Can this be vendored in instead?

Comment (aneeshkp) [reply pkg/gpsd/gpsd.go:1]: no i have modified to add  TOFF and other report in my updated PR .yet t push it

Comment (aneeshkp) [reply pkg/gpsd/gpsd.go:1]: and they are outdated. the struct values are in nano sec  not micro sec

Comment (aneeshkp) [reply pkg/gpsd/gpsd.go]: vendored via https://github.com/aneeshkp/go-gpsd

**Review by josephdrichard — APPROVED** [review]
/lgtm

**Pull request merged**